msbuild/tests/Xamarin.iOS.Tasks.Tests/TaskTests/MTouchTaskTests.cs

```
@@ -27,9 +27,20 @@ public CustomMTouchTask ()
 			AppExtensionReferences = new Microsoft.Build.Framework.ITaskItem[] { };
 		}
 
+		public string ResponseFile = "";
+
 		public new string GenerateCommandLineCommands ()
 		{
-			return base.GenerateCommandLineCommands ();
+			var cl = base.GenerateCommandLineCommands ();
+
+			try {
+				using (StreamReader sr = new StreamReader (ResponseFilePath))
+					ResponseFile = sr.ReadToEnd ();
+			} catch (Exception e) {
+				Console.WriteLine ($"The file could not be read: {e.Message}");
```

won't that hide the error ? i.e. should not this fail ?